app/views/habits/show.html.erb

```diff
@@ -28,6 +28,14 @@
 
 <% end %>
 
+
+
+<h2>Habit Completion Rate</h2>
+<%= pie_chart @completion_rate, colors: ["#78c2ad", "#f3969a"] %>
```

Comment:
Maybe you can pass the colors with variables so that when you go back to the code, you can have a general idea of what the colors could be without having to look at the app

Reply:
heard